src/entities/Profile.ts

```diff
@@ -32,6 +32,11 @@ export enum Year {
 	PHD = 'PhD Student'
 }
 
+export enum Visibility {
+	PRIVATE = 0,
+	PUBLIC = 1
```

Comment:
This is really pedantic but can these be `Private` and `Public` just to follow suit with the other enums?